ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+Have you read Lisk-JS's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/lisk-js/master/CODE_OF_CONDUCT.md
+
+### Description
+
+[Description of the issue]
+
+### Steps to Reproduce
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+**Expected behavior:** [What you expect to happen]
+
+**Actual behavior:** [What actually happens]
+
+**Reproduces how often:** [What percentage of the time does it reproduce?]
+
+### Versions
+
+Please provide the version you are working with. Please include the OS and what version of the OS you're running.
+Also please provide if you are using the NodeJS or Browser version of Lisk-JS.
```

Comment:
"Also please tell us whether you are using Lisk-JS in a browser (which one? which version?) or in Node.js."